src/frontend-scripts/components/section-main/Players.jsx

```diff
@@ -415,16 +416,20 @@ class Players extends React.Component {
 		}
 
 		const index = gameInfo.publicPlayersState.findIndex(player => player.userName === this.state.reportedPlayer);
-
-		this.props.socket.emit('playerReport', {
-			uid: gameInfo.general.uid,
-			userName: this.props.userInfo.userName || 'from replay',
-			gameType: gameInfo.general.isTourny ? 'tournament' : gameInfo.general.casualGame ? 'casual' : 'standard',
-			reportedPlayer: `${index ? `{${index + 1}} ${this.state.reportedPlayer}` : this.state.reportedPlayer}`,
-			reason: $('input[name="reason"]').attr('value'),
-			comment: this.state.reportTextValue
-		});
-		$(this.reportModal).modal('hide');
+		if (this.state.reportLength <= 140) {

```

Comment:
Heh I'll allow it but typically you wouldn't set more state, you'd just check the length of reportValue and prevent submit or prevent typing more.  But thanks!